### DISCUSSION TITLE
Can Anyone can provide the code to combine the Page Header and tabs (Top Navigation)

### DISCUSSION THREAD

**Discussion by Hopy-Jaden** [bdlukaa/fluent_ui#668] [Q&A]
![Screenshot 2023-01-01 171742](https://user-images.githubusercontent.com/96371992/210166222-070ff5e8-ac2d-49d3-a057-2b988f97057b.png)
This is an app I am working on, and the above image is something I want to make. Can anyone provide me with an example of putting the tabs of the top navigation view with the Page header in the header parameter?

![image](https://user-images.githubusercontent.com/96371992/210166389-45e3d095-2000-468c-9275-b63261697715.png)
also in the top navigation view of the documentation, it is a completely straight line. anyone knows how to fix it into a windows 11 style, with the line being rounded?

Appreciate it if you can answer it.

**Comment by bdlukaa** [comment]
I suggest you to read https://github.com/bdlukaa/fluent_ui/issues/919.

What you're trying to achieve is called Pivot and was deprecated in Windows UI 2 and is not going to be implemented in this library.
